hack/render-manifests.go

```diff
@@ -22,7 +22,7 @@ func main() {
 	}
 
 	if len(os.Args) < 6 {
-		fmt.Fprintf(os.Stderr, "usage: go run render-manifests.go [namespace] [image] [pull-policy] [input-dir] [output-dir]\n")
+		fmt.Fprintf(os.Stderr, "usage: go run render-manifests.go [namespace] [image] [pull-policy] [input-dir] [output-dir] [handler|operator]\n")
```

Comment:
Maybe you can use <> for mandatory parameter and [ ] for the last optional parameter?